### ISSUE TITLE
Work out an initial price of subscriptions

### ISSUE THREAD

**Issue by equivalentideas** [openaustralia/planningalerts#704]

**Comment by equivalentideas** [comment]
@henare and I discussed this yesterday and decided $29/month was a good starting point. We based this on no particular formula, just what seemed reasonable without appearing markedly high.

**Comment by henare** [comment]
We've talked about cohort testing of the price before. If we decide to do it or not, we should document the reasoning here.

**Comment by equivalentideas** [comment]
We did talk about trying to cohort test our way towards the most profitable price, but decided against it.

I understood that we thought we didn't have enough potential subscribers to get statistically significant data as to provide a clear result. We thought we are better off picking a price based on what we felt was reasonable and then seeing what happens once it's released.

Did I miss anything from our discussion @henare ?

I think we can then increase or decrease the price based on the response.

$29 a month feels on the average to low end of subscription services a business or individual might be used to, like an internet connection, phone plan, pay TV, industry news etc. .

As this is going from a free service for these people to a paid one, we could really jar people if it's too expensive to start off.

**Comment by henare** [comment]
Great summary @equivalentideas.

Rethinking the cohort stuff I realised we don't really need statistically significant data from releasing to cohorts - we could use it to get a sense of if something is working or not.

**Comment by equivalentideas** [comment]
Are you thinking we would:

* a) release different prices to two or more cohorts simultaneously and then pick a winner for the next round,
* b) release to one cohort, look at the results and adjust and then do another one, like we did the pricing emails earlier in the year.

or something else?

**Comment by henare** [comment]
I was thinking b) but a) would work too.

**Comment by equivalentideas** [comment]
In practice, b) would mean: we do $29 for the first bunch, then choose another price like $25 for the next. Then we might try $35. Then by the time we're rolled out completely, we'll be able to compare a bunch of different prices and we might have an idea of what the best price is.

a) would mean: we could do high and low numbers like $29 and $35 for the first one, and then if $29 is more successful try $19 and $29, and then if $29 is more successful try $25 and $29—and close in on something.

I'm really not sure, I could go either way.

Technically, we'd have to somehow flag users with a cohort number so if they came back to the subscription/new page during their trial, the price was the same—I feel like its to unexpected and weird to return to a page and the price to be different.

**Comment by henare** [comment]
>I'm really not sure, I could go either way.

Didn't we say above we've decided against cohort testing? It sounds like my comments made you have second thoughts.

**Comment by equivalentideas** [comment]
>  It sounds like my comments made you have second thoughts.

I think I'm going through our discussion yesterday because of the discomfort of the idea that we're doing all this work that isn't for our core audience, and it might not be as valuable as it could be, just because we picked the wrong price :stuck_out_tongue_closed_eyes:

The actual question that I didn't think of last night is: What happens when we want to change the price?

Do people who've see the subscription page come back to it and see a different price? I think that could damage confidence in the service. But do we care if a small group of these people who aren't our core group get turned off? Is it worth the time of making roll out more complex? I feel like it's more important to get something out to a small initial group and more forward with that feedback.

> Rethinking the cohort stuff I realised we don't really need statistically significant data from releasing to cohorts - we could use it to get a sense of if something is working or not.

Could you just give a simple example of how releasing to cohorts would help make a decision as opposed to not doing it? I think it's important we have a sense of it something's working, but I don't quite get what data we would have to go by in the different situations.

**Comment by henare** [comment]
>Do people who've see the subscription page come back to it and see a different price? I think that could damage confidence in the service.

Well then we make it so they don't come back and see another price! :)

>Could you just give a simple example of how releasing to cohorts would help make a decision as opposed to not doing it? I think it's important we have a sense of it something's working, but I don't quite get what data we would have to go by in the different situations.

I don't understand what you mean so I'll explain my quote you're responding to further: we don't need statistically significant results to get a sense if one price is resulting in lots more conversions than another one.

**Comment by equivalentideas** [comment]
:) @henare I think this is just getting more confusing. Can we work this out on a call?

**Comment by equivalentideas** [comment]
We've decided to do cohort testing, starting with two prices $15 and $34.

On the phone @henare and I came to the decision that we should do cohort testing to help us determine price, because price is a big factor in whether or not the subscription concept will be a success. Currently we don't have much information to base our price on, so if we get it wrong we wont know why. Importantly we wont know if it was the price that was wrong or something else.

If both the prices we test result in no subscriptions, we know something else is wrong.

If one price clearly wins, we know the flow is working to some extent and can test more prices to find the most profitable one.

We chose $15 and $34 as the first high/low test, because it is a wide position to narrow in from.
